### DISCUSSION TITLE
Cloud SQL Node.js Connector in Slonik

### DISCUSSION THREAD

**Discussion by manuel-valles** [gajus/slonik#560] [Q&A]
Hey community,

I am trying to implement https://github.com/GoogleCloudPlatform/cloud-sql-nodejs-connector in an existing project with Slonik and I don't find a way to pass the client options:

Example with PG (straight forward):
```
const clientOpts = await connector.getOptions({
  instanceConnectionName: 'my-project:region:my-instance',
  ipType: 'PUBLIC',
});

const pool = new Pool({
  ...clientOpts,
  user: 'my-user',
  password: 'my-password',
  database: 'db-name',
  max: 5,
});
```

I would appreciate it if I could please get some help on this. Thanks in advance 🙏

**Comment by hallvors** [comment]
Does an example using startLocalProxy() get you further? The connection URL can be passed to Slonik, I believe.
https://github.com/GoogleCloudPlatform/cloud-sql-nodejs-connector/blob/main/examples/prisma/postgresql/connect.mjs
